scavenger-schema/src/main/resources/db/changelog/db.changelog-master.yaml

```diff
@@ -1,6 +1,8 @@
 databaseChangeLog:
   - include:
       file: db/changelog/schema.sql
+  - include:
+      file: db/changelog/schema-1.1.1.sql
```

Comment:
maybe... `1.1.2`

Reply:
haha..... fixed in 51ccd12425738493b11b9706a412fdcce621657b